images/plbase/plbase-install.sh

```diff
@@ -45,45 +45,17 @@ echo "setting up postgres..."
 mkdir /var/postgres && chown postgres:postgres /var/postgres
 su postgres -c "initdb -D /var/postgres"
 
+# TODO: use standard OS Python installation? The only reason we switched to Conda
+# was to support R and `rpy2`, but now that we've removed those, we might not
+# get any benefit from Conda.
 echo "setting up conda..."
 cd /
 arch=`uname -m`
 curl -LO https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-${arch}.sh
 bash Miniforge3-Linux-${arch}.sh -b -p /usr/local -f
```

Comment:
I don't know enough to form a firm opinion on this, but it seems that conda is not used anywhere else, and the Python grader installs its own version of it, so it should be save to remove it.

Reply:
Discussion continued here: https://github.com/PrairieLearn/PrairieLearn/pull/9886#discussion_r1612128790

We're going to leave this as-is for now.